.github/workflows/build-status-check.yml

```diff
@@ -15,7 +15,7 @@ on:
 jobs:
   build_main_project:
     name: Build ${{ vars.PROJECT_NAME }}
-    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.0.1
+    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.1.0
```

Comment:
Need to add the `checkout-ref` and `checkout-repository` workflow inputs.
Make sure to use Velaptor as a guide.  We implemented it wrong with the CASL project.

Reply:
duplicated?